test/functional/people_controller_test.rb

```diff
@@ -78,18 +78,4 @@ def stub_role_appointment(role_type, options = {})
       assert_select "a[href='/government/policies/welfare-reform']", text: "Welfare reform"
     end
   end
-
```

Comment:
I think the link is still there - perhaps we don't care about testing it anymore?

Reply:
Hmm, maybe we do. I've put it back.